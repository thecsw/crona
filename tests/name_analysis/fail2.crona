main : void() {
     a : int;
     b : string;
     b = "hello";
     a = b;
}