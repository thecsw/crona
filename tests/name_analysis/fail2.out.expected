main(->void) : void(){
	a(int) : int;
	b(byte array [0]) : byte array[0];
	b(byte array [0]) = "hello";
	a(int) = b(byte array [0]);
}
